### PR TITLE
*: Copy route53 baseDomain advice to openshift-install locations

### PR DESCRIPTION
**Pull request by wking**
Builds on #351; review that first.

The old `examples/aws.yaml` location is no longer discoverable for folks using the new installer.

### CODE REVIEW

**Comment by wking** [comment]
Example prompt output:

```console
$ ./openshift-install install-config
? Email Address a@b.com
? Password [? for help] ****
? SSH Public Key <none>
???? The base domain of the cluster. All DNS records will be sub-domains of this base.

For AWS, this must be a previously-existing public Route 53 zone.  You can check for any already in your account with:

  $ aws route53 list-hosted-zones --query 'HostedZones[? !(Config.PrivateZone)].Name' --output text
? Base Domain 
```

**Comment by abhinavdahiya** [comment]
This might be useful in docs; but not sure if this is required in help for prompt.

**Comment by wking** [comment]
> This might be useful in docs; but not sure if this is required in help for prompt.

I expect folks reading environment-variable docs will be mostly power-users who have gotten tired of the interactive prompts, and the advice is most useful to fist-timers.  Also, you only see the prompt help if you hit `?`.  But I can remove it from there if you like.  Thoughts?

**Comment by dak1n1** [comment]
Hey guys, thanks so much for working on these docs. Here is some feedback, in case that's helpful.

As someone with more of a systems / Ops background, I went straight for the env variables, rather than ever wanting to be prompted. The reason for this is that I write down the settings of every install attempt when I'm figuring out new software, and I like to be able to reproduce installs with a quick copy/paste, avoiding interactive steps whenever possible.

I started at https://github.com/openshift/installer/blob/master/README.md and followed the links from there to the env variables list. So the additions you made to the [env vars list](https://github.com/openshift/installer/blob/master/docs/user/environment-variables.md) are really helpful to me.

File: docs/user/environment-variables.md [hunk 1]
```diff
@@ -4,19 +4,38 @@ The installer accepts a number of environment variable that allow the interactiv
...
-| `OPENSHIFT_INSTALL_PLATFORM`      | The platform onto which the cluster will be installed.                                      |
-| `OPENSHIFT_INSTALL_PULL_SECRET`   | The container registry pull secret for this cluster.                                        |
-| `OPENSHIFT_INSTALL_SSH_PUB_KEY`   | The SSH key used to access all nodes within the cluster. This is optional.                  |
+* `OPENSHIFT_INSTALL_BASE_DOMAIN`:
+    The base domain of the cluster. All DNS records will be sub-domains of this base.
+
+    For AWS, this must be a previously-existing public Route 53 zone.  You can check for any already in your account with:
+
+    ```sh
+    aws route53 list-hosted-zones --query 'HostedZones[? !(Config.PrivateZone)].Name' --output text",
```
Comment (dak1n1) [inline docs/user/environment-variables.md]: there's an extra `",` at the end of this line that makes the command not copy/paste-able.

Comment (wking) [reply docs/user/environment-variables.md]: > there's an extra `",` at the end

Oops, copy/paste from the Go version.  Fixed in 9324f35 -> fa7c9e3.

**Pull request merged**